var/spack/repos/builtin/packages/py-protobuf/package.py

```diff
@@ -19,6 +19,7 @@ class PyProtobuf(PythonPackage):
 
     variant("cpp", default=False, description="Enable the cpp implementation")
 
+    version("4.21.5", sha256="eb1106e87e095628e96884a877a51cdb90087106ee693925ec0a300468a9be3a")
```

Comment:
Note that this new version is likely to break many packages with open-ended version dependencies. See https://github.com/spack/spack/pull/30880 for details. This isn't a reason not to add it, just a note that we should expect to see build issue reports due to this.

Reply:
Why don't I either remove it or put a preferred on it to avoid it.

Reply:
Up to you. The packages I care about have already been fixed. Most developers have discovered this issue already and locked the maximum version in their dependency list, so packages with a new release since then that have been added to Spack shouldn't be affected. Personally, I would rather get the bug reports so we can fix this. But I realize that's not the most stable solution. Another option would be to lock all packages to `protobuf@:3` until someone tests and verifies that they work with protobuf 4.